OWML.ModHelper.Menus/ModInputCombinationElementMenu.cs

```diff
@@ -0,0 +1,168 @@
+            GameObject.Destroy(originalMenu);
+            GameObject.Destroy(_inputMenu.GetValue<Text>("_labelText").GetComponent<LocalizedText>());
+            Initialize((Menu)_inputMenu);
+        }
+
+        public void Open(string value, string comboName, IModInputCombinationMenu combinationMenu = null, IModInputCombinationElement element = null)
+        {
+            _combinationMenu = combinationMenu;
+            _element = element;
+            _comboName = comboName;
+            _inputMenu.OnPopupConfirm += OnPopupConfirm;
+            _inputMenu.OnPopupCancel += OnPopupCancel;
+            _inputMenu.OnPopupValidate += OnPopupValidate;
+
+            var message = "Press your combination";
+
+            _inputMenu.EnableMenu(true, value);
+
+            var okCommand = InputLibrary.confirm;
+            var okPrompt = new ScreenPrompt(okCommand, "OK");
+            if (_cancelCommand == null)
+            {
+                _cancelCommand = new SingleAxisCommand();
+                var cancelBindingGmpd = new InputBinding(JoystickButton.Select);
+                var cancelBindingKbrd = new InputBinding(KeyCode.Escape);
+                _cancelCommand.SetInputs(cancelBindingGmpd, cancelBindingKbrd);
+                var commandObject = new GameObject();
+                var commandComponent = commandObject.AddComponent<ModCommandUpdater>();
+                commandComponent.Initialize(_cancelCommand);
+            }
+            var cancelPrompt = new ScreenPrompt(_cancelCommand, "Cancel");
+            var resetPrompt = new ScreenPrompt("Reset");
+            _inputMenu.SetUpPopup(message, okCommand, _cancelCommand, null, okPrompt, cancelPrompt, resetPrompt);
+            _inputMenu.GetValue<Text>("_labelText").text = message;
+        }
+
+        private bool OnPopupValidate()
+        {
+            var currentCombination = _inputMenu.Combination;
+            _console.WriteLine($"Validating combination {currentCombination}");
+            var collisions = _inputHandler.GetCollisions(currentCombination);
+            if (collisions.Count > 0 && collisions[0] != _comboName)
+            {
+                _twoButtonPopup.EnableMenu(true);
+                _twoButtonPopup.SetUpPopup($"this combination collides with \"{collisions[0]}\"", InputLibrary.confirm2, null,
+                    new ScreenPrompt(InputLibrary.confirm2, "Ok"), new ScreenPrompt("Cancel"), true, false);
+                _twoButtonPopup.GetValue<Text>("_labelText").text = $"this combination collides with \"{collisions[0]}\"";
+                return false;
+            }
+            if (_combinationMenu == null)
+            {
+                return true;
+            }
+            foreach (var element in _combinationMenu.CombinationElements)
+            {
+                _console.WriteLine($"Checking against current {element.Title}");
+                if (element.Title == currentCombination && element != _element)
```

Comment:
var elements = _combinationMenu.CombinationElements
    .Where(element => element.Title == currentCombination && element != _element)
foreach (var element in elements)

Reply:
K